keyboards/angel64/rev1/matrix.c

```diff
@@ -0,0 +1,287 @@
+/*
```

Comment:
Does this actually need a custom matrix?

Reply:
This is the matrix that requires col2row and row2col.

![SnapCrab_NoName_2019-7-13_10-53-59_No-00](https://user-images.githubusercontent.com/15257475/61165513-b09a7e80-a55c-11e9-80ec-da0ef110fd06.png)